package.json

```
@@ -43,7 +43,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js icons"
   },
-  "version": "2.37.0",
+  "version": "2.37.1",
```

It's not a Vanilla code change, we don't need to bump the version.

```suggestion
  "version": "2.37.0",
```